libfio.c

```diff
@@ -353,14 +353,17 @@ int initialize_fio(char *envp[])
 	 * can run into problems on archs that fault on unaligned fp
 	 * access (ARM).
 	 */
+	compiletime_assert((offsetof(struct thread_data, ts) % 8) == 0, "ts");
```

Comment:
Actually, why isn't this aligned to begin with? We don't pack struct thread_data, so I'm puzzled why this would ever be unaligned?